cmd/server/http/http.go

```diff
@@ -728,3 +726,26 @@ func release(payload *payload, flowSvc *flow.Service) http.HandlerFunc {
 func convertTimeToEpoch(t time.Time) int64 {
 	return t.UnixNano() / int64(time.Millisecond)
 }
+
+type commitInfo struct {
+	AuthorName  string
+	AuthorEmail string
+	Service     string
+}
+
+func extractAuthorFromCommit(message string) (commitInfo, error) {
+	pattern := `^\[(?P<service>.*)\].*\nSigned-off-by:\s(?P<authorName>.*)\s<(?P<authorEmail>.*)>`
```

Comment:
This could be compiled ahead of time to reduce the need for it on every webhook.